src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -133,26 +133,47 @@
           </v-row>
         </template>
 
-        <div
-          v-if="minimumThreeDirectorError"
+        <MessageBox
+          v-if="hasAttemptedSubmission && minimumThreeDirectorError"
           id="minimum-three-director-error"
-          class="my-6"
+          class="mt-6"
+          color="red"
         >
-          <p class="error-text">
-            The business type cannot be changed. A Community Contribution Company requires a minimum of three directors.
+          <p>
+            <v-icon
+              color="red"
+              class="error-icon"
+            >
+              mdi-alert
+            </v-icon>
+            <strong class="pl-2 pt-6">Update directors</strong>
           </p>
-        </div>
+          <p class="pl-8">
+            A BC Community Contribution Company requires at least three directors.
+            File a director change and then come back and update the business type.
```

Comment:
This seems OK to me but I thought we had decided to only warn users if they had less than the minimum but allow the alteration anyway (like we do if a CCC files a Change of Directors with less than 3 directors). Please confirm with Mihai that this is OK.

Reply:
If this is turned into a warning, change the colour to gold :)

Reply:
Jacqueline will discuss this with Janis on Monday. If Janis is not available then we will make a decision anyway.

Reply:
And if you are ready to merge today then go with your existing changes.

We can create another PR next week if needed.